proto/regen/ecocredit/v1alpha1/types.proto

```diff
@@ -23,7 +23,8 @@ message ClassInfo {
   // metadata is any arbitrary metadata to attached to the credit class.
   bytes metadata = 4;
 
-  // credit_type describes the type of credit (e.g. carbon, biodiversity), as well as unit and precision.
+  // credit_type describes the type of credit (e.g. carbon, biodiversity), as
+  // well as unit and precision.
   CreditType credit_type = 5;
```

Comment:
i just realized this probably should change in v1 to just `string credit_type_abbrev` for the v4 upgrade. made an issue here https://github.com/regen-network/regen-ledger/issues/858

Reply:
oh nvm, we already have it as a string, just need to worry about the migration then